### DISCUSSION TITLE
Mulighet for å slette stoppesteder / bysykkelstasjoner fra admin panelet

### DISCUSSION THREAD

**Discussion by wfeiring** [entur/tavla#308] [Ideas]
Per i dag kan man ikke fjerne stoppesteder eller bysykkel stasjoner man har lagt til inne på admin panelet. Hvis man sitter og prøver seg fram litt kan man fort ende opp med et admin-panel som blir uoversiktlig og rotete. Det hadde vært utrolig deilig å kunne fjerne stoppesteder inne fra admin-panelet uten å måtte lage en ny tavle.

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
